app/build.gradle.kts

```diff
@@ -56,7 +56,7 @@ android {
         compose = true
     }
     composeOptions {
-        kotlinCompilerExtensionVersion = "1.5.4"
+        kotlinCompilerExtensionVersion = "1.5.15"
```

Comment:
please remove this whole block and add the compiler instead, like here; https://github.com/google-developer-training/basic-android-kotlin-compose-training-dice-roller/pull/166/files